CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Fix:  libmicrohttpd now uses poll() and not select() (#2724)
```

Comment:
Maybe epoll and https://github.com/telefonicaid/fiware-orion/issues/2166 should be mentionces here. In addition, explain the "functional" nature of the change. Suggestion:

```
- Fix:  libmicrohttpd now uses poll()/epoll() and not select() so fds greater than 1024 can be used for incomming connections  (#2724)
```